src/main/java/com/esri/samples/raster/raster_rendering_rule/RasterRenderingRuleSample.java

```diff
@@ -0,0 +1,204 @@
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.raster.ImageServiceRaster;
+import com.esri.arcgisruntime.raster.RenderingRule;
+
+public class RasterRenderingRuleSample extends Application {
+
+  private ComboBox<RenderingRuleInfo> renderingRuleInfoComboBox;
+  private MapView mapView;
+  private ArcGISMap map;
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Raster Rendering Rule Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a control panel
+      VBox controlsVBox = new VBox();
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.6)"), CornerRadii.EMPTY, Insets.EMPTY)));
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setSpacing(8);
+      controlsVBox.setMaxSize(260, 160);
+      controlsVBox.getStyleClass().add("panel-region");
+
+      // create progress indicator
+      ProgressIndicator progressIndicator = new ProgressIndicator();
+      progressIndicator.setVisible(false);
+
+      // create drop down menu of Rendering Rules
+      renderingRuleInfoComboBox = new ComboBox<>();
+      renderingRuleInfoComboBox.setPromptText("Select a Raster Rendering Rule");
+      renderingRuleInfoComboBox.setEditable(false);
+      renderingRuleInfoComboBox.setMaxWidth(260.0);
+      renderingRuleInfoComboBox.setConverter(new StringConverter<RenderingRuleInfo>() {
+        @Override
+        public String toString(RenderingRuleInfo renderingRuleInfo) {
+
+          return renderingRuleInfo != null ? renderingRuleInfo.getName() : "";
+        }
+
+        @Override
+        public RenderingRuleInfo fromString(String string) {
+          return null;
+        }
+      });
+
+      // create a label for the Rendering Rule info
+      Label renderingRuleInfoLabel = new Label("");
+      renderingRuleInfoLabel.setFont(new Font(12));
+      renderingRuleInfoLabel.setTextFill(Color.WHITE);
+      renderingRuleInfoLabel.setMaxWidth(220);
+      renderingRuleInfoLabel.setWrapText(true);
+
+      // add the ComboBox and Label to the controlsVBox
+      controlsVBox.getChildren().addAll(renderingRuleInfoComboBox, renderingRuleInfoLabel);
+
+      // create a Streets BaseMap
+      map = new ArcGISMap(Basemap.createStreets());
+
+      // add the map to a new map view
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // create an Image Service Raster as a raster layer and add to map
+      final ImageServiceRaster imageServiceRaster = new ImageServiceRaster("http://sampleserver6.arcgisonline.com/arcgis/rest/services/CharlotteLAS/ImageServer");
```

Comment:
Since you use this string twice in the code, I'd store it as a local variable. @tschie is this ok for convention?